app/assets/stylesheets/layouts/_content.scss

```diff
@@ -4,3 +4,7 @@
   @include column(12);
   margin-bottom: $baseline-unit*4;
 }
+
+.l-content {
+  @extend %l-content;
+}
```

Comment:
Now that we're not using the abstract class we should replace it with this concrete one.